src/Components/DeleteModal.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import { AiFillDelete } from "react-icons/ai";
+
+
+class DeleteModal extends Component {
+    constructor(props) {
+        super(props)
+    
+        this.state = {
+            showModal: false
+        }
+        this.setShowModal = this.setShowModal.bind(this)
+        this.onDelete = this.onDelete.bind(this)
+    }
```

Comment:
User arrow methods instead of binding, see some video tutorial for  more clarification